### ISSUE TITLE
VueJS Series

### ISSUE THREAD

**Issue by duttakapil** [coderplex/meetups#2]
We're planning to do a series of meetups on VueJS, lead by Praveen. Let's discuss here possible ideas for the content of the meetups, format, agenda, venue details etc and schedule them. 

These can be planned both online and/or offline

**Comment by praveenpuglia** [comment]
- Vue Fundamentals + Vue Route 
  * Installing, Editor Setup
  * Vue Instance & different parts of it.
  * Vue Components
  * Vue CLI
  * Basic Routing

- Vuex
  * Understanding the basics
  * Understanding the flow, Actions, Mutations, Getters etc.
  * Demo app + best practices.

**Comment by duttakapil** [comment]
So these are two separate sessions correct? 
Meetup 1 :
Topic : Vue Fundamentals + Vue Route
Date : 16th December
Time : 3 pm - 5 pm
Venue : Pramati Technologies
Format : ?

Meetup 2 : 
Topic : Vuex
Date : 23rd December
Venue : ?
Format : ?

**Comment by praveenpuglia** [comment]
Sounds good to me! Preparing for the first one. I am working on the segments for the talk and demos. Will update all of that here.

**Comment by duttakapil** [comment]
Share your slides and github repo links here when they are ready @praveenpuglia

**Comment by duttakapil** [comment]
Online Web Cast / Meetup 
Date : 8th or 9th Jan, 2018
Topic : ?
Time : ?
Speaker : @praveenpuglia

**Comment by vijay-teddy** [comment]
hey @praveenpuglia are we doing the web cast / meetup for 9th?

**Comment by praveenpuglia** [comment]
@vijay-teddy nope. we aren't doing it. Unfortunately I forgot that I have additional duties for this week at day job. I'll update here as an when I can.

Also, it would be great if i can get some suggestions on what topics would people like to have a webcast on.. Just concepts or building something live?

**Comment by vijay-teddy** [comment]
@praveenpuglia people have answered and everyone wants building something live.
